translation/h/list/00_00/personality_voice_c-1_01_00/translation.txt

```diff
@@ -10,11 +10,11 @@
 ちょっと恥ずかしいわ……=I'm a little embarrassed...
 もう、何とか言ってよ=Come on, say something.
 早く手を出してきて……=Your hand, hurry...
-くちゅびる……やぁらかぃ……（唇、柔らかい）=... Mmlps...ss fft... (Your lips, so soft)
-もっと……ちょうらい……（もっとちょうだい）=Dmmphpmh...mmmmr... (Deeper, more)
+くちゅびる……やぁらかぃ……（唇、柔らかい）=... Mmlps... ss fft... (Your lips, so soft)
```

Comment:
Is the stuff in `()` just what they're saying without the sluring? Should it be directly translated or used at a guide?